.travis.yml

```diff
@@ -1,18 +1,10 @@
 language: scala
 scala:
   -  2.12.4
-
-addons:
-apt:
-  sources:
-    - google-chrome
-  packages:
-    - google-chrome-stable
-
 before_install:
```

Comment:
it seems like you accidentally chose the wrong version during your merge, please correct that, we want to use chrome and execute the unit tests.